### DISCUSSION TITLE
Changed the owner of my Plex server, now Overseerr won't connect to Plex

### DISCUSSION THREAD

**Discussion by kylerm42** [sct/overseerr#3066] [Help]
After the Plex breach, I decided to secure things a bit by having a separate account as server owner from the one my family uses everywhere. Ever since then, Overseerr can't connect to Plex. No servers show up when refreshing the list, and after filling in the info for a manual connection, clicking "Save Changes" results in a "Failed to connect to Plex" message. This is in the logs for all connection tests and scans: `{"errorMessage":"Permission denied even after attempted authentication :( Wrong username and/or password maybe?"}`.

I did just create another fresh instance of Overseerr, and was able to set things up just fine, everything worked as expected. I can use that, if necessary, but was hoping not to lose all my request history, if possible. So is there a way to essentially reset everything with Plex on an existing instance of Overseerr? Thanks for any help!

**Comment by valfur03** [comment]
I just had the same issue. I had to switch the accounts into the database by hand. This seems to work, let's wait a couple of days.

Here is a small summary of what I did:

1. backup Overseerr data
2. connect to the database using sqlite CLI
3. list the two accounts I want to switch (the old owner, and the new one)
    ```sql
    SELECT * FROM "user" WHERE id = <user_id>;
    ```
4. switch info for both accounts (email must be unique, so you might want to use a placeholder value during the operation)
    ```sql
    UPDATE "user" SET email = <email>, username = <username>, plexId = <plexId>, plexToken = <plexToken>, avatar = <avatar>, password = <password>, plexUsername = <plexUsername>, resetPasswordGuid = <resetPasswordGuid>, recoveryLinkExpirationDate = <recoveryLinkExpirationDate>, movieQuotaLimit = <movieQuotaLimit>, movieQuotaDays = <movieQuotaDays>, tvQuotaLimit = <tvQuotaLimit>, tvQuotaDays = <tvQuotaDays> WHERE id = <user_id>;
    ```

You can see that the user IDs are not kept by their actual users, so it means that anything that belonged to user1, now belongs to user2. It is not possible to change that because Overseerr seems to get the "Owner" flag from the user with ID 1.

A procedure in the web UI would be greatly appreciated though, and if possible with distinction of the owner and the user ID...

**Reply by CircuitQuest** [reply]
Thanks for the pointer, I just did something similar on Jellyseerr.
But instead of updating the info like, this I tried just swapping the ID's around.


E.g. assuming only two existing users:
```sql
UPDATE "user" SET id = 3 WHERE id = 1;
UPDATE "user" SET id = 1 WHERE id = 2;
UPDATE "user" SET id = 2 WHERE id = 3;
```
Also had to set the `permissions` field to "2":
```sql
UPDATE "user" SET permissions = 2 WHERE id = 1;
```
For the other account I found it easier to fix the permissions in the UI afterwards, but if you know what value you need you can update that as well.

And then it should be working, but as always YMMV :)
